src/Approvals/SFA.DAS.Approvals/Application/Apprentices/Queries/GetReviewApprenticeshipUpdates/GetReviewApprenticeshipUpdatesQueryHandler.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.Approvals.Enums;
+using SFA.DAS.Approvals.Extensions;
+using SFA.DAS.Approvals.InnerApi;
+using SFA.DAS.Approvals.InnerApi.CommitmentsV2Api.Requests;
+using SFA.DAS.Approvals.InnerApi.CommitmentsV2Api.Responses;
+using SFA.DAS.Approvals.Services;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Extensions;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Approvals.Application.Apprentices.Queries.GetReviewApprenticeshipUpdates
+{
+    public class GetReviewApprenticeshipUpdatesQueryHandler : IRequestHandler<GetReviewApprenticeshipUpdatesQuery, GetReviewApprenticeshipUpdatesQueryResult>
+    {
+        private readonly ICommitmentsV2ApiClient<CommitmentsV2ApiConfiguration> _apiClient;
+        private readonly IProviderStandardsService _providerStandardsService;
+        private readonly ServiceParameters _serviceParameters;
+
+        public GetReviewApprenticeshipUpdatesQueryHandler(
+            ICommitmentsV2ApiClient<CommitmentsV2ApiConfiguration> apiClient,
+            IProviderStandardsService providerStandardsService,
+            ServiceParameters serviceParameters)
+        {
+            _apiClient = apiClient;
+            _providerStandardsService = providerStandardsService;
+            _serviceParameters = serviceParameters;
+        }
+
+        public async Task<GetReviewApprenticeshipUpdatesQueryResult> Handle(GetReviewApprenticeshipUpdatesQuery request, CancellationToken cancellationToken)
+        {
+            var apprenticeshipTask = _apiClient.GetWithResponseCode<GetApprenticeshipResponse>(new GetApprenticeshipRequest(request.ApprenticeshipId));
+
+            var updatesTask = _apiClient.GetWithResponseCode<GetApprenticeshipUpdatesResponse>(new GetApprenticeshipUpdatesRequest(request.ApprenticeshipId, Convert.ToByte(ApprenticeshipStatus.WaitingToStart)));
```

Comment:
I'm not sure I understand this filter on apprenticeship status - it must surely be the status of the _update_ that matters? Also, the byte conversion could probably be done within the request class itself and not here, as it looks pretty untidy.